cibyl/sources/zuul/api.py

```diff
@@ -332,6 +314,17 @@ def jobs(self):
         """
         raise NotImplementedError
 
+    @abstractmethod
+    @deprecated(details="Access builds through jobs instead.")
```

Comment:
Maybe it's a not genius question but why do we wanna have deprecated code instead of deleting it?

Reply:
Because I have not taken the time to see if it is in use in some parts of the code. I just to do it this way for now.

Reply:
If all of we agree to do it at this way then okay. Thx.